packages/amplify_analytics_pinpoint/android/src/main/kotlin/com/amazonaws/amplify/amplify_analytics_pinpoint/AmplifyAnalyticsPinpointPlugin.kt

```diff
@@ -67,6 +55,24 @@ public class AmplifyAnalyticsPinpointPlugin : FlutterPlugin, ActivityAware, Meth
         when (call.method) {
             "addPlugin" ->
                 AmplifyAnalyticsBridge.addPlugin(result, context)
+            "startSession" -> {
+                // Hack: The AutoSessionTracker in the Pinpoint plugin listens for lifecycle changes and
```

Comment:
We shoud change this terminlogy to say with a 'TODO' what a better solution would entail

Reply:
Fixed